tests/Queue/QueueSizeTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Queue;
+
+use Illuminate\Bus\Queueable;
+use Orchestra\Testbench\TestCase;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class QueueSizeTest extends TestCase
+{
+    public function test_queue_size()
+    {
+        \Queue::fake();
```

Comment:
This code isn's going to work. There's no such class.

Reply:
You'll need to reference the facade class.

Reply:
@GrahamCampbell, then how on earth this is working locally and on travisCI?
```
vendor/bin/phpunit tests/Queue/QueueSizeTest.php 
PHPUnit 7.5.15 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.1.25 with Xdebug 2.5.3
Configuration: /Users/j/Documents/Sponge/snaptest/2/framework/phpunit.xml.dist

.                                                                   1 / 1 (100%)

Time: 146 ms, Memory: 12.00 MB

OK (1 test, 4 assertions)
```

Reply:
Probably because the class alias is setup.